WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -184,27 +226,53 @@ private extension CardPresentPaymentsOnboardingUseCase {
     }
 
     func isCountrySupported(countryCode: String) -> Bool {
-        return Constants.supportedCountryCodes.contains(countryCode)
+        return Constants.WCPay.supportedCountryCodes.contains(countryCode)
```

Comment:
nit: what if `Constants.WCPay.supportedCountryCodes` differs from `Constants.Stripe.supportedCountryCodes`? when do we check for `Constants.Stripe.supportedCountryCodes`?